nvgpu/__main__.py

```diff
@@ -23,7 +26,16 @@ def main():
             gpus = gpus[:args.limit]
         print(','.join(gpus))
     elif args.command == 'list':
-        pretty_list_gpus()
+        if args.loop:
+            while True:
+                try:
+                    pretty_list_gpus()
+                    time.sleep(args.loop)
+                except KeyboardInterrupt:
+                    break
+        else:
+            pretty_list_gpus()
+
```

Comment:
Cannot be the same effect achieved by using `watch` command?

Reply:
@pinkavaj You are right. We can do the same with `watch`. I just like the quick commands like `nvidia-smi -l`.